FF118 mathvariant not global (#29238)
mdn/content: files/en-us/learn/mathml/first_steps/text_containers/index.md

</math>
```

{{ EmbedLiveSample('Automatic italicization of <mi>', 700, 50) }}

> **Note:** [This table from MathML Core](https://w3c.github.io/mathml-core/#italic-mappings) provide the exhaustive list of characters that are subject to italicization, together with the corresponding italic characters.

## Reverting automatic italicization of \<mi>

> It is sometimes needed to revert this default italic transformation. For that purpose, just attach a `mathvariant="normal"` attribute on the `<mi>` element.

Author: ```suggestion
In order to revert this default italic transformation you can attach a `mathvariant="normal"` attribute on the `<mi>` element.
```
